pootle/templates/admin/language_team.html

```diff
@@ -0,0 +1,132 @@
+{% extends "admin/base_app.html" %}
```

Comment:
It should be `{% extends "languages/base.html" %}`.  Also place it within `templates/languages/admin/`. See https://github.com/translate/pootle/blob/master/pootle/templates/languages/admin/characters.html for expected sibling template.